### PR TITLE
UIView extension memory leak fix

### PR DESCRIPTION
**Pull request by kirakik**
Fixed potential memory leak of addSubviews UIView extension method

- [x] New Extension
- [ ] Changed more than one extension, but all changes are related
- [x] Trivial change (doesn't require changelog)

### CODE REVIEW

File: Sources/UIViewExtensions.swift [hunk 1]
```diff
@@ -31,8 +31,8 @@ extension UIView {
 
     /// EZSwiftExtensions, add multiple subviews
     public func addSubviews(_ views: [UIView]) {
-        views.forEach { eachView in
-            self.addSubview(eachView)
+        views.forEach { [weak self] eachView in
```
Comment (Khalian) [inline Sources/UIViewExtensions.swift]: Good catch. I presume this implies the subview has a reference to the parent view and vice versa and due to ARC (Automated ref counting) there is no way to resolve the ref cycle without a weak ref.

Comment (kirakik) [inline Sources/UIViewExtensions.swift]: Exactly. I was going to change it for a `for...in...` loop, but I decided against it, didn't want to change too much here

**Comment by Khalian** [comment]
@kirakik You can change the internals without breaking the API contract. As long as all unit tests for this pass through, it should be fine (although I dont remember if it has any).

File: Sources/UIViewExtensions.swift (hunk 1 above)
Comment (furuyan) [inline Sources/UIViewExtensions.swift]: `forEach` is not `@escaping` method.  So i think this case is not need to describe `weak self`.

`public func forEach(_ body: (Element) throws -> Void) rethrows`

Comment (goktugyil) [inline Sources/UIViewExtensions.swift]: @furuyan What exactly are you recommending?

**Pull request merged**